exporters/BxDRobotExporter/robot_exporter/JointResolver/EditorsLibrary/EditLimits.cs

```diff
@@ -222,6 +215,20 @@ private void Angular_End_CheckedChanged(object sender, EventArgs e)
                 }
         }
 
+        private void Linear_Current_textbox_ValueChanged(object sender, EventArgs e)
+        {
+            var pos = Convert.ToDouble(Linear_Current_textbox.Text) * 2.54;
+            ((InventorSkeletalJoint)joint).GetWrapped().asmJointOccurrence.DriveSettings.StartValue = pos + " cm";
+            ((InventorSkeletalJoint)joint).GetWrapped().asmJointOccurrence.DriveSettings.EndValue = pos+1 + " cm";
+            ((InventorSkeletalJoint)joint).GetWrapped().asmJointOccurrence.DriveSettings.GoToStart();
```

Comment:
Using the `GoToStart()` method for animations doesn't seem like the proper way to do these single-position animations, but I couldn't find an alternative. Maybe someone who knows the Inventor API has a better solution?